--- mbedtls-1.3.12/include/polarssl/config.h
+++ mbedtls-1.3.12/include/polarssl/config.h
@@ -105,7 +105,7 @@
  *
  * Comment if your system does not support time functions
  */
-#define POLARSSL_HAVE_TIME
+//#define POLARSSL_HAVE_TIME
 
 /**
  * \def POLARSSL_HAVE_IPV6
@@ -2058,7 +2058,7 @@
  *
  * Requires: POLARSSL_SSL_CACHE_C
  */
-#define POLARSSL_SSL_CACHE_C
+//#define POLARSSL_SSL_CACHE_C
 
 /**
  * \def POLARSSL_SSL_CLI_C
